OpenKh.Kh2/Motion.cs

```diff
@@ -66,10 +118,186 @@ public class RawMotion
+            [Data] public float BoundingBoxMaxX { get; set; }
+            [Data] public float BoundingBoxMaxY { get; set; }
+            [Data] public float BoundingBoxMaxZ { get; set; }
+            [Data] public float BoundingBoxMaxW { get; set; }
+            [Data] public float FrameLoop { get; set; }
+            [Data] public float FrameEnd { get; set; }
+            [Data] public float FramePerSecond { get; set; }
+            [Data] public float FrameCount { get; set; }
+            [Data] public int UnknownTable1Offset { get; set; }
+            [Data] public int UnknownTable1Count { get; set; }
+            [Data] public int Unk98 { get; set; }
+            [Data] public int Unk9c { get; set; }
+        }
+
+        public class InterpolatedMotion
+        {
+            public short BoneCount { get; set; }
+            public int TotalFrameCount { get; set; }
+            public int Unk48 { get; set; }
+            public int Table8Offset { get; set; }
+            public int Table7Offset { get; set; }
+            public int Table7Count { get; set; }
+            public int Table6Offset { get; set; }
+            public int Table6Count { get; set; }
+            public float BoundingBoxMinX { get; set; }
+            public float BoundingBoxMinY { get; set; }
+            public float BoundingBoxMinZ { get; set; }
+            public float BoundingBoxMinW { get; set; }
+            public float BoundingBoxMaxX { get; set; }
+            public float BoundingBoxMaxY { get; set; }
+            public float BoundingBoxMaxZ { get; set; }
+            public float BoundingBoxMaxW { get; set; }
+            public float FrameLoop { get; set; }
+            public float FrameEnd { get; set; }
+            public float FramePerSecond { get; set; }
+            public float FrameCount { get; set; }
+            public int Unk98 { get; set; }
+            public int Unk9c { get; set; }
+
+            public List<InitialPoseTable> StaticPose { get; set; }
+            public List<BoneAnimationTable> ModelBoneAnimation { get; set; }
+            public List<BoneAnimationTable> IKHelperAnimation { get; set; }
+            public List<TimelineTable> Timeline { get; set; }
+            public List<IKChainTable> IKChains { get; set; }
+            public List<IKHelperTable> IKHelpers { get; set; }
+            public List<JointTable> Joints { get; set; }
+            public FooterTable Footer { get; internal set; }
+        }
+
+        public class InitialPoseTable
+        {
+            [Data] public short BoneIndex { get; set; }
+            [Data] public short Channel { get; set; }
+            [Data] public float Value { get; set; }
+
+            public override string ToString() =>
+                $"{BoneIndex} {Transforms[Channel]} {Value}";
+        }
+
+        public class BoneAnimationTable
+        {
+            [Data] public short JointIndex { get; set; }
+            [Data] public byte Channel { get; set; }
```

Comment:
This `Channel` is bit fielded according to revel8n's feedback:

```
    uint8  keyType  : 4;
    uint8  preType  : 2;
    uint8  postType : 2;
```

![image](https://user-images.githubusercontent.com/5955540/101271255-97831500-37c4-11eb-8ead-f580cc376e7b.png)

Reply:
Ok thanks, I will update code and docs following your feedback. What is the purpose of KeyCount?